lms/djangoapps/grades/forms.py

```diff
@@ -0,0 +1,40 @@
+"""
+Defines a form for providing validation of subsection grade templates.
+"""
+import logging
```

Comment:
Question: will django allow us to put all admin-related code in a separate `admin` folder?
The folder would then have the admin-specific models and these forms.